### ISSUE TITLE
Some translations have a Placeholder Renderer in Phrase class

### ISSUE THREAD

**Issue by ilnytskyi** [magento/magento2#23661]
### Preconditions (*)

Magento 2.3.1 probably other versions as well
1. Mostly happens in custom controllers or modules.
2. However the exception is thrown by core class
`\Magento\Framework\App\Request\CsrfValidator::createException`

![Selection_076](https://user-images.githubusercontent.com/3192770/61026969-367bc580-a3b5-11e9-83e5-743d4d284ea4.png)

### Steps to reproduce (*)

0. Use different language in store
1. Trigger any error to see "**Invalid Form Key. Please refresh the page.**" e.g. upload a file with a size bigger than PHP allows via custom form, make sure the controller has CSRF protection.
2. On front see that the message is not translated.
3. The same happens during store emulation e.g. when cron sends emails in emulation loop or renders blocks based on **store_id**

### Expected result (*)

1. The class `Phrase` has composite renderer 
`\Magento\Framework\Phrase\Renderer\Composite`
2. All strings put into **__()** or **new Phrase()** are translated 

### Actual result (*)

1. The class `Phrase` has placeholder renderer
2. In some cases not all string are translated

Workaround:
Set **composite** renderer manually before a string will be processed.

```
\Magento\Framework\Phrase::setRenderer(
    \Magento\Framework\App\ObjectManager::getInstance()
        ->get(\Magento\Framework\Phrase\Renderer\Translate::class)
);
```

UPD: in some cases it does not help since the area is not initialized 
`\Magento\Framework\App\AreaInterface`
`\Magento\Framework\App\Area`

UPD2:
looks like cookie string are translated only when, inline translation is enabled see
`\Magento\Theme\Controller\Result\MessagePlugin::setCookie`
```
if ($this->inlineTranslate->isAllowed()) {
                foreach ($messages as &$message) {
                    $message['text'] = $this->convertMessageText($message['text']);
                }
            }
```

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Alfa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-18666` were created

**Issue Available**: @engcom-Alfa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by norgeindian** [comment]
This issue occurs for example in custom console commands. The composite renderer is not set, so the placeholder renderer is used and nothing gets translated.

**Comment by bartoszkubicki** [comment]
Still an issue in 2.3.5. Any updates?

**Comment by magento-engcom-team** [comment]
Hi @ilnytskyi. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-23896` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.

**Comment by convenient** [comment]
I believe this is the commit here that fixes this https://github.com/magento/magento2/commit/14558892cc0aa06629e0e7b0a78276587fcc6d6d
